index.html

```diff
@@ -3096,6 +3096,12 @@ <h3 id="applying">
           consistent.
         </p>
         <p>
+          To allow the community can easily find extensions, please add your
+          extensions to the <a href=
+          "https://github.com/w3c/manifest/wiki/Extensions-Registry">Extensions
+          Registry</a>.
```

Comment:
No. "standard" ones could be more "experimental" than proprietary things, and vice versa: the point is to distinguish things that a group of implementers are working on with the aim to standardize, and things that ship by a single vendor but there might not be any appetite to standardize for whatever reason. 

Having said that, for each category, those could be marked as stable or experimental.